### DISCUSSION TITLE
Encrypting Secret Data at Rest

### DISCUSSION THREAD

**Discussion by rikirolly** [kubernetes-sigs/cluster-api#8185] [Q&A] [answered]
I am trying to use AWS KMS provider for data encryption on pure EC2 cluster (without EKS).
I understood I need to start aws-encryption-provider pod as static pod but I don't know how to accomplish this with Cluster-API.
Anyone can help me?

**Comment by sbueringer** [comment, marked as answer]
This question might be better asked in the cluster-api-provider-aws repository
